app/src/common/shared/com/igalia/wolvic/ui/widgets/dialogs/RestartDialogWidget.java

```diff
@@ -31,7 +31,7 @@ public void updateUI() {
 
             } else if (index == PromptDialogWidget.POSITIVE) {
                 mWidgetManager.saveState();
-                postDelayed(() -> SystemUtils.scheduleRestart(getContext(), 100), 500);
+                postDelayed(() -> SystemUtils.restart(getContext()), 500);
```

Comment:
Any reason for increasing the delay to half a second?

Reply:
It's not increasing the delay. We had a delay of half a second originally.

Reply:
Oh indeed, I got confused by our scheduled call.